src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -1236,6 +1317,18 @@ void SolverBase::cleanup( real64 const GEOS_UNUSED_PARAM( time_n ),
                           DomainPartition & GEOS_UNUSED_PARAM( domain ) )
 {
   m_solverStatistics.outputStatistics();
+
+  if( getLogLevel() > 0 )
+  {
+    for( auto & timer : m_timers )
+    {
+      real64 const time = std::chrono::duration< double >( timer.second ).count();
+      real64 const minTime = MpiWrapper::min( time );
+      real64 const maxTime = MpiWrapper::max( time );
+      if( maxTime > 0 )
+        GEOS_LOG_RANK_0( GEOS_FMT( getName() + ": {} time = {} s (min), {} s (max)", timer.first, minTime, maxTime ) );
```

Comment:
Consider making solver name part of the format string, also adding missing braces
```suggestion
      if( maxTime > 0 )
      {
        GEOS_LOG_RANK_0( GEOS_FMT( "{}: {} time = {} s (min), {} s (max)", getName(), timer.first, minTime, maxTime ) );
      }
```